### ISSUE TITLE
Use actionbarextras to set regular menu items

### ISSUE THREAD

**Issue by Gtwin** [ricardoalcocer/actionbarextras#51]
i was just wandering if it is possible to use actionbarextras to set regular menu items. 
I wanted to use FontAwesome to set icon actions by changing the font. 
thanks for your incredible work, this module is very usefull :)

**Comment by manumaticx** [comment]
I'm just thinking about how to implement this. Would it make sense to provide a method to change the font for all menu items? Or would it be better to add the menu item through ABX? Something like:
```javascript
activity.onCreateOptionsMenu = function(e){
  abx.addMenuItem({
     menu: e.menu,
     title: 'some option',
     font: 'myFont.otf'
  });
};
```
Another solution would be to enhance existing menuItems like:
```javascript
activity.onCreateOptionsMenu = function(e){
  var menuItem = e.menu.add ({
     title: 'some option'
  });

  abx.setMenuItem({
    item: menuItem,
    font: 'myFont.otf'
  });
};
```

Personally, I like the first one better but to follow the concept of enhancing we may should go with the second one. This would you also give the possibility to change multiple menu items at once.

I'm not yet sure if it works as fine as with the title / subtitle font or if it will work with FontAwesome, but I'll try asap.

**Comment by manumaticx** [comment]
Closing this since this is resolved with #60 thanks to @mpociot
